HOWTO.md

```diff
@@ -0,0 +1,10 @@
+# How to experiment ComputePressure API with Chrome
+
+The ComputePressure API is currently available only in Chrome Canary behind a flag.
+
+1) Download a version of Chrome with ComputePressure implemented ([Desktop](https://www.google.com/intl/en_ie/chrome/canary/)).
+2) Launch Chrome with the command line flag --enable-features=ComputePressure.
+2) Navigate to `chrome://flags` and enable `Experimental Web Platform features`.
+3) Navigate to a test page, e.g. https://wicg.github.io/compute-pressure/demo/.
```

Comment:
it's not an example :-) so no need for e.g., i.e. would make more sense - or just leave it out